golang-rock/1.22-24.04/rockcraft.yaml

```diff
@@ -0,0 +1,94 @@
+name: golang
+build-base: ubuntu@24.04
+base: bare
+version: '1.22'
+summary: The Go programming language
+description: |
+  Go is an open source programming language that enables the production of
+  simple, efficient and reliable software at scale.
+platforms: 
+  amd64:
+  arm64:
+
+# We are not using the _daemon_ run-user, since rockcraft adds this user
+# locked. We however needs sudo to run apt.
+# https://github.com/canonical/rockcraft/blob/acc25f85af3ec691eb7f9f41558164745c03e028/rockcraft/oci.py#L297
+
+parts:
+  golang:
+    plugin: nil
+    source: https://github.com/zhijie-yang/chisel-releases.git
+    source-type: git
+    source-branch: ROCKS-2299/golang-24.04-simplified
+    override-build: |
+      arch="${CRAFT_ARCH_TRIPLET_BUILD_FOR//_/-}"
+      chisel cut --release ./ --root ${CRAFT_PART_INSTALL} \
+          base-files_base \
+          base-files_chisel \
+          base-files_release-info \
+          ca-certificates_data \
+          golang_cgo-support \
+          binutils-${arch}_assembler \
+          binutils-${arch}_linker \
+          cpp-13-${arch}_native \
+          gcc-13-${arch}_gcc-13
+
+      find ${CRAFT_PART_INSTALL}/usr/share/go-1.22 -depth \( \
```

Comment:
this is out of date, no? i think the version in #4 has one more clause" `! -path '*src/runtime/synctest*'`

Reply:
also i'd add the same comment line as in #4: `# Remove the unnecessary test files`.